src/actions/classifier.js

```diff
@@ -72,6 +72,13 @@ export function saveAnnotation(task, value) {
   }
 }
 
+export function removeAnnotationValue(task, value) {
+  return (dispatch, getState) => {
+    const workflowID = getState().classifier.currentWorkflowID
+    dispatch(removeState(`classifier.annotations.${workflowID}.${task}`, value))
```

Comment:
Oh, do you mean there can only be one value allowed at a time?  It allows multiples right now.  For example like this: (Nothing Here, nothing Here either are two unlinked tasks)
[{"task":"T1","task_label":"Is this a **scary** bug?","value":["Nothing Here","Nothing Here either"]},{"task":"T0","task_label":"Is this a **scary** bug?","value":"No"}]